### PR TITLE
Update manage-connections-from-windows-operating-system-components-to-microsoft-services.md

### PR DESCRIPTION
**Pull request by starchturrets**
## Why
Add instruction to disable Widgets using Group Policy, instead of only the registry.

- Closes #[Issue Number]

### CODE REVIEW

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 1]
```diff
@@ -1916,9 +1916,13 @@ Add a REG_DWORD value named **DisableOneSettingsDownloads** to **HKEY_LOCAL_MACH
 
 Widgets is a news and feeds service that can be customized by the user. If you turn off this service, apps using this service may stop working.
 
-You can turn off Widgets by setting the following registry entries:
+You can turn off Widgets by:
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:1919]:
```suggestion
To turn off Widgets:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @starchturrets, kindly commit the minor suggested change. Many thanks.

**Comment by starchturrets** [comment]
Sorry, this is the first time I've used git in a while to collaborate with others, how do I do that?

**Comment by JohanFreelancer9** [comment]
> Sorry, this is the first time I've used git in a while to collaborate with others, how do I do that?

You should see a button that says "Commit suggestion". Just select it and you are done.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by DHB-MSFT** [comment]
@JohanFreelancer9 Please don't review, edit, or approve any pull requests for content in the Windows "privacy" folder. We have a separate internal process for reviewing & approving updates to privacy content. Thank you.

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 Please don't review, edit, or approve any pull requests for content in the Windows "privacy" folder. We have a separate internal process for reviewing & approving updates to privacy content. Thank you.

@DHB-MSFT My apologies. Thank you for pointing it out to me. I will not do so in future.

**Comment by DHB-MSFT** [comment]
@starchturrets Thanks for your suggestion to update/improve this content. We've published some changes to the article based on your suggestion.

**Pull request closed without merging**